feature/google/src/test/java/BillingWrapperTest.kt

```diff
@@ -168,77 +154,52 @@ class BillingWrapperTest {
     }
 
     @Test
-    fun defersCallingSkuQueryUntilConnected() {
-        mockStandardSkuDetailsResponse()
+    fun defersCallUntilConnected() {
```

Comment:
some existing tests were "general behavior of BillingWrapper", i.e. about the connection logic. most of these used `querySkuDetailsAsync` as their example call into `BillingWrapper`, but which method was called didn't seem to matter. so to keep those tests in the main `BillingWrapperTest`, i used `consumePurchaseAsync` -- that method is exactly the same between BillingClient versions